packages/connectors/src/metaMask.ts

```diff
@@ -57,6 +57,7 @@ export class MetaMaskConnector extends InjectedConnector {
           if (ethereum.isOkxWallet || ethereum.isOKExWallet) return
           if (ethereum.isOneInchIOSWallet || ethereum.isOneInchAndroidWallet)
             return
+          if (ethereum.isOneKey) return
```

Comment:
Does OneKey override MetaMask? If not, please remove this change.

Reply:
We provided two Provider api.
+ window.ethereum
+ window.$onekey.ethereum

We originally wanted to use window.$onekey.ethereum, Because window.ethereum has Settings that you can turn off.